index.html

```diff
@@ -526,4 +642,4 @@ <h2 class="blog__title col">
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
can you add an empty line at the end here?

Reply:
I think I fixed it but I'm not sure, the file I'm editing has an empty line at the bottom :-\